### ISSUE TITLE
"Size Overflow" error when executing OrtCreateTensorWithDataAsOrtValue()

### ISSUE THREAD

**Issue by wm6057** [microsoft/onnxruntime#1321]
Hi guys,

I have a Keras-Tensorflow based model and have converted to Onnx format.

When I'm trying to load it inside a **MFC

** project, "**Size Overflow**" error just came out.

But when I load the same Onnx model  in python by **onnxruntime.InferenceSession()**, it works!

The Model output Dim is :
Input 0 : type = 1	
Input 0 : num_dims = 4	
Input 0 : dim 0 = -1	
Input 0 : dim 1 = 128	
Input 0 : dim 2 = 128
Input 0 : dim 3 = 3	

These are how I tried to load an image as an input data in C environment (Please also tell me what kind of loading function is correct):

1. Take every pixel's RGB (sorted as BGR) and then push_back into a 1-D <float>vector. "**Size Overflow**" error showed.

2. Take every pixel's RGB (sorted as BGR) and then push_back into a 4-D <float>vector to make the input data just look like what I formed in python environment as (1, 128, 128, 3). "**Size Overflow**" error showed too.

**System information**

- Windows 10
- ONNX Runtime installed from binary
- ONNX Runtime version: v0.4.0
- Visual Studio version: 2017
- CPU

Does anyone know what I have done in C environment is wrong?

Really appreciate for any suggestions!

**Comment by bitnick10** [comment]
` input_node_dims[0] = 1;`
